src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -145,7 +153,24 @@ struct OperationViewModel {
         self.requiredQueryParams = requiredQueryParams
         self.requiredHeaders = requiredHeaders
         self.uriParams = uriParams
+        self.clientMethodOptionsViewModel = ClientMethodOptionsViewModel(
```

Comment:
We shouldn't add `viewModel` to the property name. It just makes the the code verbose and we don't do that anywhere else that we have a view model property inside another view model.

Reply:
Good catch. Renamed.